spot-client/src/spot-remote/ui/views/help.js

```diff
@@ -67,11 +68,41 @@ class Help extends React.Component {
                             Continue
                         </Button>
                     </div>
+                    {
+                        isSpotControllerApp()
+                            ? null
+                            : this._renderLegalLinks()
+                    }
                 </div>
             </View>
         );
     }
 
+    /**
+     * Creates links to legal documents related to using Spot.
+     *
+     * @private
+     * @returns {ReactElement}
+     */
+    _renderLegalLinks() {
+        return (
+            <div className = 'legal-links'>
+                <a
+                    href = 'https://www.8x8.com/terms-and-conditions/privacy-policy'
```

Comment:
Can't these be configurable, like on Jitsi Meet?

Reply:
You think we should?

Reply:
The Jitsi "Spot" app would have different links. Yes, they may point to the same place, but we redirect from jitsi.org.

Reply:
I sense you do want is so I'm going to make it configurable in a follow up commit today, probably the first thing I work on

Reply:
Thank you! That shouldn't block this PR though.